app/src/main/java/fr/free/nrw/commons/CommonsApplication.java

```diff
@@ -91,6 +91,7 @@
     @Override
     public void onCreate() {
         super.onCreate();
+        ACRA.init(this);
```

Comment:
Acra is also inited on line number 122.

Reply:
Do you think moving it fixes the missing dialogs? If so, I'll remove the one on 122 which I missed.

Reply:
Played around a bit: using the initAcra method does not work.

Reply:
Yeah, having this Acra.init() line results in the dialog asking for feedback again after a crash. Using the initAcra() after the  onCreate does not fix it.

Reply:
Great! In that case lets remove initAcra function. Also, can you try sending logs to check if the reports are getting generated correctly